src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetPermissionsRequest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.SharedOuterApi.InnerApi.Requests;
+
+public class GetPermissionsRequest : IGetApiRequest
+{
+    public string GetUrl => $"permissions?Ukprn={Ukprn}&PublicHashedId={PublicHashedId}";
+
+    public long? Ukprn { get; }
+
+    public string? PublicHashedId { get; set; }
+
+    public GetPermissionsRequest(long? ukprn, string? publicHashedId)
+    {
+        Ukprn = ukprn;
+        PublicHashedId = publicHashedId;
+    }
+}
+
+public enum Operation : short
```

Comment:
This enum is used in Response, but is hidden in Request object file. 

Also I think we should have a folder within Requests and Response to separate objects out which related to ProviderRelationships

Reply:
moved to own class in repsonses